src/content/i18n/fr.yml

```
@@ -0,0 +1,119 @@
+a11y.sectionLink: Titre de la section
+# Site settings
+site.title: Documentation Astro
+# Left Sidebar
+leftSidebar.sponsoredBy: Sponsorisé par
+# Right Sidebar
+rightSidebar.community: Communauté
+rightSidebar.joinDiscord: Rejoindre notre Discord
+rightSidebar.contribute: Contribuer
+rightSidebar.translatePage: Traduire cette page
+# Footer
+footer.privacyPolicy: Politique de Confidentialité
+# Used in `<Since>`: Added in: v0.24.0 [NEW]
+since.addedIn: 'Ajouté à la version :'
+since.new: Nouveau
+since.beta: Bêta
+# Installation Guide
+install.autoTab: "Automatiquement via l'ILC"
+install.manualTab: Configuration manuelle
+# `<DeployGuidesNav>` vocabulary
+deploy.sectionTitle: Guides de déploiement
+deploy.altSectionTitle: Plus de guides de déploiement
+deploy.ssrTag: SSR
+deploy.staticTag: Statique
+# CMS Guides vocabulary
+cms.navTitle: Plus de guides sur les CMS
+# Media Guides vocabulary
+media.navTitle: Plus de guides sur les DAM
+# Migration Guides vocabulary
+migration.navTitle: Plus de guides sur les migrations
+# `<RecipeLinks>` vocabulary
+recipesLink.singular: 'Méthode associée :'
+recipesLink.plural: Méthodes associées
+# 404 Page
+404.title: Page introuvable
+404.content: Cette page ne fait pas partie de notre système solaire.
+404.linkText: Ramenez moi à la maison
+# Integrations vocabulary
+integrations.changelog: Journal des modifications
+integrations.footerTitle: "Plus d'intégrations"
+integrations.renderers: "Framework d'interface utilisateur"
+integrations.adapters: Adaptateurs SSR
+integrations.others: Autres intégrations
+integrations.more: "Plus d'intégrations"
+# Checklist component
+checklist.or: ou
+# Multiple Choice component
+multipleChoice.defaultCorrect: Correct !
+multipleChoice.defaultIncorrect: Réessayer !
+multipleChoice.submitLabel: Envoyer
+# Tutorial Progress
+progress.todo: À faire
+progress.done: Terminer
+# Tutorial Navigation
+tutorial.trackerLabel: Suivi du tutoriel
+tutorial.unit: Unité
+# Tutorial
+tutorial.getReady: Préparez-vous à…
+# Feedback Fish widget
+feedback.button: Laissez un commentaire
+feedback.a11yLabel: Formulaire de commentaires
+feedback.formTitle: Comment pouvons-nous vous aider ?
+feedback.categoryGroupLabel: Choisissez le type de commentaire
+feedback.issue: Problème
+feedback.createIssue: Créer une issue GitHub
+feedback.createIssue.description: Le moyen le plus rapide d'alerter notre équipe dun problème.
```

missing apostrophe (the last one apparently!):
```suggestion
feedback.createIssue.description: Le moyen le plus rapide d'alerter notre équipe d'un problème.
```